docs/client_reference.rst

```diff
@@ -30,8 +30,8 @@ Usage example::
 
 
 .. class:: ClientSession(*, connector=None, loop=None, request_class=None,\
-                          response_class=None, cookies=None, headers=None,\
-                          auth=None)
+                          response_class=None, ws_response_class=None,\
+                          cookies=None, headers=None, auth=None)
```

Comment:
Got it. These params will not include the (optional) flag correct? Because some class must be passed as arg for each (although it is usually the default) . Like such: 

   :param request_class: Request class implementation. ``ClientRequest``
                         by default.

   :param response_class: Response class implementation.
                          ``ClientResponse`` by default.

   :param ws_response_class: WebSocketResponse class implementation.
                             ``ClientWebSocketResponse`` by default.